evalml/tests/pipeline_tests/test_component_graph.py

```diff
@@ -1685,6 +1685,30 @@ def test_describe_component_graph(return_dict, example_graph, caplog):
         assert component.name in out
 
 
+def test_describe_component_graph_TypeError():
```

Comment:
Nitpick: function names should be exclusively snake case - `typeerror` or `type_error` is just fine in this case.